core/themes/basis/theme-settings.php

```diff
@@ -69,4 +69,87 @@
   foreach ($fields as $field) {
     $form['footer'][$field] = color_get_color_element($form['theme']['#value'], $field, $form);
   }
+
+  $theme_name = $form['theme']['#value'];
+  $theme_path = backdrop_get_path('theme', 'basis');
+  require_once $theme_path . '/template.php';
+  $install_version = config_get('system.core', 'install_version');
+
+  $css_update_options = array(
+    'default' => t('Default'),
+    'all_updates' => t('All updates'),
+    'custom' => t('Specific version'),
+  );
+
+  $form['css_updates'] = array(
+    '#type' => 'details',
+    '#open' => FALSE,
+    '#summary' => t('CSS Updates'),
+    '#details' => t('There are occasionally updates available for Basis that may affect existing sites, changing this setting may cause unexpected changes to your theme. For more information and details, see the <a href="https://docs.backdropcms.org/documentation/layouts-and-templates" target="_blank">online documentation</a>.</br></br>Select which updates you would like to accept. '),
+    '#attributes' => array(
+      'class' => array('description'),
+    ),
+  );
+
+  // Load the saved configuration value.
+  $form['css_updates']['css_update'] = array(
+    '#title' => t('CSS Update Options'),
+    '#type' => 'radios',
+    '#options' => $css_update_options,
+    '#default_value' => theme_get_setting('css_update', $theme_name),
+    '#config' => 'basis.settings',
+    'default' => array(
+      '#description' => t('Accept no updates introduced after installation.'),
+    ),
+    'all_updates' => array(
+      '#description' => t('Warning: Some future changes may effect your site.'),
+    ),
+    'custom' => array(
+    ),
+  );
+
+  // Custom select css update versions.
+  $options = basis_css_versions_updated();
```

Comment:
Add the no updates option here instead, so it can be translated and be given a different key (an empty string):

```suggestion
  $options = backdrop_map_assoc(basis_css_versions_updated());
  $options[''] = t('No updates');
```